tests/integration/test_worker.py

```diff
@@ -215,3 +215,13 @@ def test_sigusr1(self):
             worker.expect('Consumer started')
             worker.send_signal(signal.SIGUSR1)
             worker.expect('traceback.format_stack')
+
+    def test_batch(self):
+        """Batch tasks are run"""
+        li = [tasks.echo.subtask(args=("foo %i" % i, )) for i in range(2)]
```

Comment:
If there wasn't a host parameter I would do the same with couldn't come up with better solution :(